query/src/main/scala/filodb/query/exec/MetadataRemoteExec.scala

```diff
@@ -68,6 +68,7 @@ case class MetadataRemoteExec(queryEndpoint: String,
         case _: MetadataMapSampl         => mapTypeQueryResponse(response, id)
         case _: LabelCardinalitySampl    => mapLabelCardinalityResponse(response, id)
         case _: TsCardinalitiesSampl     => mapTsCardinalitiesResponse(response, id)
+        case _: TsCardinalitiesSamplV2   => mapTsCardinalitiesResponseV2(response, id)
```

Comment:
Ask about the need of filodb dataset. Is it needed after reducing between raw and downsample data in TsCardReduceExec ?